common-app-lib/src/cmr/common_app/services/ingest/subscription_common.clj

```diff
@@ -1,20 +0,0 @@
-(ns cmr.common-app.services.ingest.subscription-common
```

Comment:
unused?

Reply:
cmr.common-app.services.ingest.subscription-common moved to common.util